ts/screens/authentication/CieSuccessScreen.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import WaitComponent from "../../components/screens/cie/WaitComponent";
+
+type OwnProps = {
+  navigation: NavigationScreenProp<NavigationState>;
+};
+
+type Props = OwnProps;
+
+class CieSuccessScreen extends React.Component<Props> {
```

Comment:
Since the _**WaitComponent**_ is still small you can include it inside this screen.
So **_WaitComponent_** could be deleted